loopchain/peer/consensus_siever.py

```diff
@@ -94,7 +96,7 @@ def __build_candidate_block(self, block_builder, next_leader, vote_result):
         if not vote_result:
             raise NotEnoughVotes
 
-        self._block_manager.get_blockchain().add_block(block, vote)
+        self._block_manager.get_blockchain().add_block(block, vote.votes)
```

Comment:
@windies21 @hyerim-kim @zsaladin 

in `blockchain.py`,
```
    def add_block(self,
                  block: Block,
                  confirm_info=None,
                  need_to_write_tx_info=True,
                  need_to_score_invoke=True) -> bool:
```
`add_block` has confirm_info as second parameter.

if `vote.votes` is confirm_info, it would be better to add comment above.